app/controllers/predicates/AuthoriseAsAgentOnly.scala

```diff
@@ -52,35 +53,35 @@ class AuthoriseAsAgentOnly @Inject()(enrolmentsAuthService: EnrolmentsAuthServic
       case Some(affinityGroup) ~ allEnrolments =>
         (isAgent(affinityGroup), allEnrolments) match {
           case (true, _) =>
-            logger.debug("[AuthoriseAsAgentOnly][invokeBlock] - Is an Agent, checking HMRC-AS-AGENT enrolment")
+            infoLog("[AuthoriseAsAgentOnly][invokeBlock] - Is an Agent, checking HMRC-AS-AGENT enrolment")
             checkAgentEnrolment(allEnrolments, block)
           case (_, _) =>
-            logger.debug("[AuthoriseAsAgentOnly][invokeBlock] - Is NOT an Agent, rendering Technical Difficulties view")
+            errorLog("[AuthoriseAsAgentOnly][invokeBlock] - Is NOT an Agent, rendering Technical Difficulties view")
             Future.successful(errorHandler.showInternalServerError)
         }
       case _ =>
-        logger.warn("[AuthoriseAsAgentOnly][invokeBlock] - Missing affinity group")
+        errorLog("[AuthoriseAsAgentOnly][invokeBlock] - Missing affinity group")
         Future.successful(errorHandler.showInternalServerError)
     } recover {
       case _: NoActiveSession =>
-        logger.debug("[AuthoriseAsAgentOnly][invokeBlock] - No Active Session, rendering Session Timeout view")
+        errorLog("[AuthoriseAsAgentOnly][invokeBlock] - No Active Session, rendering Session Timeout view")
         Unauthorized(sessionTimeoutView())
       case _: AuthorisationException =>
-        logger.warn("[AuthoriseAsAgentOnly][invokeBlock] - Authorisation Exception, rendering Technical Difficulties view")
+        errorLog("[AuthoriseAsAgentOnly][invokeBlock] - Authorisation Exception, rendering Technical Difficulties view")
         errorHandler.showInternalServerError
       case error: UpstreamErrorResponse =>
-        logger.warn(s"[AuthoriseAsAgentOnly][invokeBlock] - Upstream error response received: ${error.message}")
```

Comment:
warn

Reply:
why should this be warn? its an upstream error